crates/matrix-sdk-crypto/src/machine.rs

```diff
@@ -3241,6 +3243,9 @@ pub(crate) mod tests {
             get_machine_pair_with_setup_sessions_test_helper(alice_id(), user_id(), false).await;
         let room_id = room_id!("!test:example.org");
 
+        let mut room_keys_withheld_received_stream =
+            Box::pin(bob.store().room_keys_withheld_received_stream());
```

Comment:
You may want to use [`pin_mut!`](https://docs.rs/futures-util/latest/futures_util/macro.pin_mut.html) here, as we do elsewhere in the code. It does a similar thing but without boxing the value.